### DISCUSSION TITLE
Is there any built in method to skip duplicate data

### DISCUSSION THREAD

**Discussion by indahud** [supabase/supabase#1834] [Questions] [answered]
I am trying insert some data. I want to skip the data if their unique keys are same. Prisma provides `skipDuplicates: true` to skip the data. Is there any similar way in supabase for this?

**Comment by steve-chavez** [comment, marked as answer]
Hey @indahud,

Our REST API supports doing a "skipDuplicates" but this is not yet exposed in client libraries. I've added the feature request on https://github.com/supabase/postgrest-js/issues/189.

**Reply by rosswaycaster** [reply]
PR up to implement `ignoreDuplicates`! https://github.com/supabase/postgrest-js/pull/195
